content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md

```diff
@@ -0,0 +1,238 @@
++++
+title = "Sigstore’s `cosign` and `policy-controller` with GKE, Artifact Registry and KMS"
+date = "2023-01-27"
+tags = ["sigstore","security","kubernetes", "kms"]
+draft = false
+author = "Mathieu Benoit"
+type = "post"
+
++++
+
+As soon as I came back from [KubeCon NA 2022](https://www.youtube.com/playlist?list=PLj6h78yzYM2O5aNpRM71NQyx3WUe1xpTn), my first ever in-person KubeCon, I felt re-energized. What a community, full of people eager to share knowledge and expertise with each others, so inspiring. I mostly attended sessions about security best practices for containers and Kubernetes (that’s what excites me these days!). Secure Software Supply Chain (S3C) was almost mentioned everywhere, for good reasons.
+
+Sigstore as a new standard for signing, verifying and protecting software, got its first own [SigstoreCon](https://www.youtube.com/playlist?list=PLj6h78yzYM2MUNId2hvHBnrGCCbmou_gl) as co-located event and [hit the General Availability (GA) milestone](https://openssf.org/press-release/2022/10/25/sigstore-announces-general-availability-at-sigstorecon/). It piqued my curiosity. I wanted to give [Sigstore’s cosign in Kubernetes clusters](https://docs.sigstore.dev/cosign/overview/#kubernetes-integrations) a try.
```

Comment:
I would hesitate for us to call ourselves a "standard", as we're definitely not one (yet). 😄

Reply:
I'm guessing this is coming straight from the main website at https://www.sigstore.dev/ :
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/1714486/216378166-85184686-0464-46a6-9ed3-036384bdfd79.png">

Reply:
Yep, exactly, @trevrosen do you still want me to replace `standard` in the context of this blog post?

Reply:
Fair point - we should probably have a look at that wording too! 😄

Reply:
I'd forgotten that that's on the front page. 😅 I'll work through my questions around that elsewhere.

Reply:
Out of curiosity, anything missing on my end to get your approval on this PR @trevrosen?